decred/decred/dcr/wire/wire.py

```diff
@@ -96,6 +96,25 @@
 # extended Decred script.
 DefaultPkScriptVersion = 0x0000
 
```

Comment:
Noting here that **dcrd** is on `ProtocolVersion = 7`, while we're on `ProtocolVersion = 6`. Will need to swing back and see what's needed to bump our `ProtocolVersion`.